frontend/src/metabase/ui/components/inputs/NumberInput/NumberInput.tsx

```diff
@@ -5,10 +5,10 @@ import type {
 import type { ChangeEvent, FocusEvent, Ref } from "react";
 import { forwardRef, useLayoutEffect, useState } from "react";
 
-import type { NumberValue } from "metabase/querying/filters/hooks/use-number-filter";
-
 import { TextInput } from "../TextInput";
 
+type NumberValue = number | "";
```

Comment:
I'm curious what the purpose of this change is, since on master the imported `NumberValue` type is defined like this, in `frontend/src/metabase/querying/filters/hooks/use-number-filter/types.ts`:

```
export type NumberValue = number | "";
```

Is the goal to avoid circular imports or something like that?

Reply:
I cherry pick it from https://github.com/metabase/metabase/pull/53647#event-16293347374

Reply:
merged to master now and this change should be gone